### PR TITLE
add post about jsonlite

### PR DESCRIPTION
**Pull request by jeroenooms**
Announcement of jsonlite 1.2

### CODE REVIEW

File: _posts/technotes/2017-01-03-jsonlite-12.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+---
+name: jsonlite-12
+layout: post
+title: "Update jsonlite 1.2"
+authors:
+  - name: Jeroen Ooms
+    url: https://ropensci.org/about/
```
Comment (sckott) [inline _posts/technotes/2017-01-03-jsonlite-12.md]: @jeroenooms you don't need to include a URL - we parse the name field and include link automatically https://github.com/ropensci/roweb/blob/master/_layouts/post.html#L14 for ropensci people

File: _posts/technotes/2017-01-03-jsonlite-12.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+# Unload jsonlite 1.1 (might need restart R on windows)
+unloadNamespace("jsonlite")
+library.dynam.unload('jsonlite', find.package('jsonlite'))
+
+# Test with jsonlite 1.2
+devtools::install_github("cran/jsonlite@1.2")
+microbenchmark::microbenchmark(jsonlite::fromJSON(json), times = 50)
+```
+
+On my Macbook this has has reduced the median time from approx 0.91s to 0.76s.
```
Comment (sckott) [inline _posts/technotes/2017-01-03-jsonlite-12.md]: duplicate `has`

**Comment by sckott** [comment]
@jeroenooms can you elaborate a bit more on some of the changes, e.g., maybe just say what `read_json` and `write_json` are for ?

And do you want the benchmark results rendered?  Seems like you would, but maybe not.

**Comment by jeroenooms** [comment]
@sckott OK the post has been updated and I bumped the date. Thanks for the feedback!

**Pull request merged**